cl/alerts/tasks.py

```diff
@@ -337,35 +339,30 @@ def send_unsubscription_confirmation(
 
 @app.task()
 def send_docket_alert_webhooks(
-    d_pk: int,
-    since: datetime,
-    docket_entries: list[DocketEntry] = None,
+    docket_entries: list[DocketEntry],
+    da_recipients: list[DocketAlertRecipient],
 ) -> None:
     """POSTS the DocketAlert to the recipients webhook(s)
 
-    :param d_pk: The Docket primary key
-    :param since: Start time for querying the docket entries
-    :param docket_entries: A list of docket entries used if we need to send a
-    webhook for a recap.email user independently
+    :param docket_entries: The new docket entries to send.
+    :param da_recipients: A list of DocketAlertRecipients objects to send the
+    webhook to.
     :return: None
     """
 
-    if docket_entries is None:
-        docket_entries = DocketEntry.objects.filter(
-            date_created__gte=since, docket_id=d_pk
-        )
-    if len(docket_entries) == 0:
-        # No new docket entries.
-        return
-
-    webhook_recipients = User.objects.filter(
-        docket_alerts__docket_id=d_pk,
-        docket_alerts__alert_type=DocketAlert.SUBSCRIPTION,
-    ).distinct()
-
+    # Only send the webhook event to users for whom this isn't their first
+    # notification for the case or if it's, only sends it if auto_subscribe is
+    # turned on.
```

Comment:
I don't understand this fix. Let's talk about it.

Reply:
Yeah of course.

Reply:
Done, now we get the webhook recipients within `get_docket_alert_recipients` as we talked about.